woocommerce/woocommerce
# [Accessibility] Hide zoomed product images for screen readers.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #44478  .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new store and import the sample products.
2. Go to Media -> Library and add an alt text to all Hoodie images.
3. Go to the Hoodie product page on the frontend www.example.com/product/hoodie/.
4. Open the Develop Tools.
5. Open the Lighthouse tab and select:
    - Mode = Navigation.
    - Device = Mobile
    - Categories = Accessibility
      ![image](https://github.com/user-attachments/assets/501b4cc2-da6a-46f0-b795-09262f5470e2)
6. Click on the "Analyze page load" button.
7. Once the report completes, check the `Values assigned to role="" are not valid ARIA roles` error.
8. Open the error details and verify that the images of the Hoodie product are not part of the error.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
